Tools/deafrica_tools/coastal.py

```diff
@@ -33,486 +18,21 @@
-    print(
-        f"Creating reduced resolution {resolution} x {resolution} "
-        f"{crs_units} tide modelling array"
-    )
-    buffered_geobox = ds.odc.geobox.buffered(buffer)
-    rescaled_geobox = GeoBox.from_bbox(bbox=buffered_geobox.boundingbox, resolution=resolution)
-    rescaled_ds = odc.geo.xr.xr_zeros(rescaled_geobox)
-
-    # Flatten grid to 1D, then add time dimension
-    flattened_ds = rescaled_ds.stack(z=(x_dim, y_dim))
-    flattened_ds = flattened_ds.expand_dims(dim={"time": time_coords.values})
-
-    # Model tides for each timestep
-    model = "FES2014" if "model" not in model_tides_kwargs else model_tides_kwargs["model"]
-    print(f"Modelling tides using {model} tide model")
-    tide_df = model_tides(
-        x=flattened_ds[x_dim],
-        y=flattened_ds[y_dim],
-        time=flattened_ds.time,
-        epsg=ds.odc.geobox.crs.epsg,
-        **model_tides_kwargs,
+def model_tides(*args, **kwargs):
+    raise ImportError(
+        "The `model_tides` function has been removed and is no longer available in this package.\n"
+        "Please install and use the `eo-tides` package instead:\n"
+        "https://geoscienceaustralia.github.io/eo-tides/migration/"
     )
 
-    # Rename x and y coordinates to match satellite array
-    tide_df = tide_df.rename({"x": x_dim, "y": y_dim}, axis=1)
 
-    # Insert modelled tide values back into flattened array, then unstack
-    # back to 3D (y, x, time)
-    tides_lowres = (
-        # Convert dataframe to xarray format
-        tide_df.set_index([x_dim, y_dim], append=True)
-        .to_xarray()
-        # Re-index and transpose back into 3D
-        .tide_m.reindex_like(rescaled_ds)
-        .transpose("time", y_dim, x_dim)
-        .astype(np.float32)
+def pixel_tides(*args, **kwargs):
+    raise ImportError(
+        "The `pixel_tides` function has been removed and is no longer available in this package.\n"
+        "Please install and use the `eo-tides` package instead:\n"
+        "https://geoscienceaustralia.github.io/eo-tides/migration/"
     )
 
-    # Optionally calculate and return quantiles rather than raw data
-    if calculate_quantiles is not None:
-
-        print("Computing tide quantiles")
-        tides_lowres = tides_lowres.quantile(q=calculate_quantiles, dim="time")
-        reproject_dim = "quantile"
-
-    else:
-        reproject_dim = "time"
-
-    # Ensure CRS is present
-    tides_lowres = tides_lowres.odc.assign_crs(ds.odc.geobox.crs)
-
-    # Reproject each timestep into original high resolution grid
-    if resample:
-
-        print("Reprojecting tides into original array")
-        tides_highres = parallel_apply(
-            tides_lowres,
-            reproject_dim,
-            odc.algo.xr_reproject,
-            ds.odc.geobox.compat,
-            resample_method,
-        )
-
-        return tides_highres, tides_lowres
-
-    else:
-        print("Returning low resolution tide array")
-        return tides_lowres
-
 
 def tidal_tag(
```

Comment:
Hey @vikineema, there is a new version of the tide tagging function in `eo-tides` too:
https://geoscienceaustralia.github.io/eo-tides/api/#eo_tides.eo.tag_tides

Example notebook: https://geoscienceaustralia.github.io/eo-tides/notebooks/Satellite_data/#using-tag_tides

Migration guide: https://geoscienceaustralia.github.io/eo-tides/migration/